src/oscar/apps/basket/abstract_models.py

```diff
@@ -772,6 +772,9 @@ def quantity_without_offer_discount(self, offer):
     def is_available_for_offer_discount(self, offer):
         return self.consumer.available(offer) > 0
 
+    def quantity_matches_offer_condition(self, offer):
```

Comment:
The name of this method is not clear to me - could you clarify what its return value represents?

Reply:
The return value represents the line's quantity that is available for the offer - probably `quantity_available_for_offer` is a better name. Thanks!